chingu-coders/Voyage2-Turtles-02
# Stylistic Changes

## Purpose
* Fix styles with quicknotes to improve UX

## Approach/Fixes
* Break line at word instead of character for longer quicklink titles
* Adjusted margin/padding/height of quiclink `li` so that hover effect doesn't crowd other links
* Darkened background of all feature containers

## Testing
* Download this PR to your local device
* Checkout the quicklinks and make sure they still work properly
* Links should now open directly instead of into a new tab
